src/app/zones/views/ZoneDetails/ZoneDetails.test.tsx

```diff
@@ -58,14 +57,9 @@ describe("ZoneDetails", () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(
-      wrapper
-        .findWhere(
-          (node) =>
-            node.type() === "button" && node.text() === Labels.EditButton
-        )
-        .exists()
-    ).toBe(true);
+
+    const editButtons = screen.getAllByRole("button", { name: "Edit" });
```

Comment:
It's good practice to use the string from `Labels.EditButton` to make it easier to maintain these tests (i.e. we can update the label without breaking tests) as well as being more reliable when doing a test for `.not.toBeInTheDocument();`